### DISCUSSION TITLE
Custom searchbar inside MenuList

### DISCUSSION THREAD

**Discussion by ihor-zinchenko** [JedWatson/react-select#5741] [General]
I need to move Search Input inside MenuList, i found this: https://codesandbox.io/s/react-select-input-in-menulist-forked-w9hw65?file=/example.js:0-3965 what works really great but it doesnt work in the latest versions of react.

How i can do the same thing? Now if i add input into MenuList i can't place cursor inside it.

**Comment by manjushsh** [comment]
Hi @ihor-zinchenko, The example you shared had missing ```getClassNames``` assignment. [Please check this fixed sandbox which works on latest react versions](https://codesandbox.io/s/react-select-input-in-menulist-forked-tqhxgt?file=/example.js).
